src/io/functionSignatures.json

```diff
@@ -0,0 +1,20 @@
+{
+"readCbModel":
```

Comment:
From what I understood, a function can have multiple signatures, which could (in future) releases be used by Matlab to e.g. display different input argument combinations. 
While there is currently no reason for it, I thought to include it for potential future releases.
Admittedly, since functionSignatures is an undocumented feature, I'm not sure what mathworks will do with it in the future. But if you look at $MATLABROOT/toolbox/shared/io/functionSignatures.json there is also a duplicate function signature for detectImportOptions.
As I said, I'm not sure, what mathworks is going to do with it, but I doubt it will hurt.